scripts/process/cells_from_mask.R

```diff
@@ -1,28 +1,29 @@
 #' take an sf object polygon and make a grid out of it
 #' 
 #' @param viz a vizlab object including a \code{cell_size} parameter input
-#' and a polygon or multipolygon \code{clip_poly}
+#' and a polygon or multipolygon \code{mask_poly}
```

Comment:
I think this switch to "mask_poly" is right in the sense that this _thing_ is a mask. The function is using that mask to clip the grid.